src/environments/gridworlds.jl

```diff
@@ -0,0 +1,26 @@
+export RLBaseGridWorld
+import .GridWorlds
+
+mutable struct RLBaseGridWorld{T<:GridWorlds.AbstractGridWorld} <: RLBase.AbstractEnv
```

Comment:
Better to follow the naming convention of other environments. I'd suggest `GridWorldEnv`.